registry/txt.go

```diff
@@ -199,8 +200,13 @@ func newPrefixNameMapper(prefix string) prefixNameMapper {
 }
 
 func (pr prefixNameMapper) toEndpointName(txtDNSName string) string {
-	if strings.HasPrefix(txtDNSName, pr.prefix) {
-		return strings.TrimPrefix(txtDNSName, pr.prefix)
+	log.Debugf("TXT record is %s", txtDNSName)
```

Comment:
Please use one line instead of having 2 log.Debugf

Reply:
I removed it the debug output.